src/app/components/trade/LiquidityDetails.tsx

```diff
@@ -551,9 +551,16 @@ const WithdrawModal = ({
     const [independentToken, dependentToken] =
       independentField === Field.CURRENCY_A ? [pair.token0, pair.token1] : [pair.token1, pair.token0];
 
+    const independentAmount = tryParseAmount(typedValue, independentToken);
+    const dependentAmountFrac = independentAmount?.multiply(price) ?? FRACTION_ZERO;
+
     parsedAmount = {
-      [independentField]: tryParseAmount(typedValue, independentToken),
-      [dependentField]: tryParseAmount(typedValue, dependentToken)?.multiply(price),
+      [independentField]: independentAmount,
```

Comment:
What is the change for?

Reply:
This fixes the issue.

`tryParseAmount(typedValue, dependentToken)?.multiply(price)` is weird, since `typedValue` is for `independentToken`. It caused the decimal of `independentToken` to be applied instead of the correct one.